src/test/codewhisperer/service/keyStrokeHandler.test.ts

```diff
@@ -213,6 +213,62 @@ describe('keyStrokeHandler', function () {
             await KeyStrokeHandler.instance.processKeyStroke(mockEvent, mockEditor, mockClient, config)
             assert.ok(invokeSpy.called)
         })
+
+        it('Should skip invoking if there is immediate right context on the same line and not a single }', async function () {
+            const casesForSuppressTokenFilling = [
+                {
+                    fileContent: 'function addTwo',
+                    line: 1,
+                    char: 9,
+                    shouldInvoke: false,
+                },
```

Comment:
just for my clarification on the test cases, for this one, it's like `{function addTwo` and the cursor is located between char `n` and char space ` ` right?

Reply:
and keyStrokeHandler shouldn't proceed the invocation since next token is space ` `

Reply:
the selection is at line 1 char 9 so it would be `function {addTwo`, it should not trigger because the `addTwo` is immediate right context and same line

Reply:
I updated the tests to be easier to understand